### DISCUSSION TITLE
我想把文档放在文件夹里面访问如何访问

### DISCUSSION THREAD

**Discussion by Lew14310** [element-plus/element-plus#19383] [Q&A]
比如我文档应该是http://localhost:3000/ss   但是会被顶掉变成http://localhost:3000/en-us导致无法访问  有大佬指明一下在哪里修改可以解决问题么

**Comment by rzzf** [comment]
这里有一些说明供你参考：[vitepress issue](https://github.com/vuejs/vitepress/issues/4886)
